### DISCUSSION TITLE
Codespace was automatically deleted and can't be recreated

### DISCUSSION THREAD

**Discussion by fugufish** [community/community#26828] [Codespaces]
<p>between 11:30 am Mountain and now a codespace created for a public project on an organization which I created was deleted. when I try to recreate the codespace I get a 404 after clicking “Create”.</p>

**Comment by fugufish** [comment]
<p>Follow up: I can create code spaces on my personal user account. But not on this organization (I am the admin on the organization)</p>

**Comment by maiertech** [comment]
<p>Until yesterday evening, I was able to create codespaces for repos owned by an org as long as I was within the 2 codespace limit. I was able to do this since I joined the beta about a week ago. Starting today, this is not possible any longer. I am redirected to a 404 page, see screenshot. I can still create codespaces for personal repos, but not for orgs.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c8f8e728dc4e436c0362c46e1bac9768f231db0b.jpeg" data-download-href="/uploads/short-url/sFSLvpTrH41hkhD6vnErn25ThG3.jpeg?dl=1" title="Screen Shot 2020-09-30 at 3.57.39 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181100207-fc383545-701d-43cc-bc0b-60d3062208a7.jpeg" alt="Screen Shot 2020-09-30 at 3.57.39 PM" data-base62-sha1="sFSLvpTrH41hkhD6vnErn25ThG3" width="690" height="394" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c8f8e728dc4e436c0362c46e1bac9768f231db0b_2_690x394.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/c/c8f8e728dc4e436c0362c46e1bac9768f231db0b_2_1035x591.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/c/c8f8e728dc4e436c0362c46e1bac9768f231db0b_2_1380x788.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c8f8e728dc4e436c0362c46e1bac9768f231db0b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-09-30 at 3.57.39 PM</span><span class="informations">1718×982 267 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by fugufish** [comment]
<p>Sounds about right. Unfortunately for me I had committed a bunch but failed to push, lost about two hours of work :frowning:</p>

**Comment by matthewisabel** [comment]
<p>We believe we know why codespaces aren’t being displayed in this case and are resulting in 404s. Will circle back shortly but we are actively working on solving this.</p>

**Comment by fugufish** [comment]
<p>are you saying that they still exist? because if that’s the case it’d save me a lot of lost work. I was stupid and didn’t push my commits as often as I should have</p>

**Comment by jkeech** [comment, marked as answer]
<p><a class="mention" href="/fugufish">@fugufish</a> – yes, they should still exist and should become visible again once the fix is deployed.</p>

**Comment by AzizAlqasem** [comment]
<p>Sounds good! I have exactly the same issue, and lost my work that has not been pushed.</p>
<p>Best,</p>

**Comment by fugufish** [comment]
<p>Looks like it has been updated and resolved.</p>

**Comment by matthewisabel** [comment]
<p>Beat me to the update <a class="mention" href="/fugufish">@fugufish</a>. Yes this has been fixed.</p>

**Comment by ktisakib** [comment]
I having the same problem my all the codespaces on my org have been deleted

**Reply by otter-computer** [reply]
Hi @ktisakib, the original content of this post was in 2020 so it's unlikely you're experiencing the same issue. Can you share some more information on exactly what happened so we can look into it? You may also have better results by contacting [GitHub Support](https://support.github.com/request) directly as they will be able to look into issues and share information directly related to your account.

**Reply by ktisakib** [reply]
# This is from the log I got and it was automatic

![image](https://user-images.githubusercontent.com/71189014/205267474-ae6d89b9-1a0c-49ef-9f5d-19ab2d4b130e.png)
